src/addons/messagelog/messagelog-addon/build.gradle

```diff
@@ -21,7 +21,7 @@ jar {
 
 tasks.register('copyGpg', Copy) {
     from 'src/test/gpg'
-    into "${layout.buildDirectory}/gpg"
+    into layout.buildDirectory.dir("gpg")
```

Comment:
Nit: Now that no string interpolation is needed anymore, the string should use single-quotes instead of double-quotes. Similar in more places below.